src/components/Searchbar.js

```diff
@@ -34,26 +43,34 @@ const Searchbar = () => {
         setQuery(event.target.value)
     }
 
+    const handleClickSong = event => {
+        const songItem = event.target.getAttribute("data-index");
```

Comment:
This use of the html5 data- being combined with JSX  takes me back to Unit I- OOP Javascript.--circa Lambda School's training kit annexe.  Like I stepped in a time machine. Makes me happy.  🚀 ⌛